installation-bundle/src/Config/ParameterDumper.php

```diff
@@ -38,13 +47,13 @@ public function __construct(string $rootDir, Filesystem $filesystem = null)
         $this->filesystem = $filesystem ?: new Filesystem();
         $parameters = [];
 
-        foreach (['app/config/parameters.yml.dist', 'app/config/parameters.yml'] as $file) {
+        foreach (self::CONFIG_FILES as $file) {
```

Comment:
I'm not sure about the logic here. Do we really want to parse all four files given they exist? If there is a `/config` folder, we should ignore the `app/config` folder IMHO.

Reply:
I thought about this as well. But the install tool will write the parameters.yml to the root on the next use, but your custom `config.yml` will not be moved.
I also thought about whether the `parameters.yml.dist` is still supported or a leftover from the pre-managed time…

Reply:
I think its a left-over from the pre-flex area in Symfony, therefore we should not support it in the `config` folder.

> But the install tool will write the parameters.yml to the root on the next use, but your custom `config.yml` will not be moved.

If there is a `config` folder, we can assume that the config files have already been moved, can't we?

Reply:
No, the `installation-bundle` will create that folder, it now writes to `/config` instead of `app/config`. We could only write to the new folder if it exists though 🤔

Reply:
Yes, that sounds reasonable.